src/core/requests.ts

```diff
@@ -68,15 +69,33 @@ namespace Requests {
 
   export const autocompleteProducts = (state: Store.State): QueryTimeProductSearchConfig => {
     const config = Selectors.config(state);
-    return Requests.chain(config.autocomplete.defaults.products, {
+    let request = {
       ...Requests.search(state),
       refinements: [],
       skip: 0,
       sort: undefined,
       language: Autocomplete.extractProductLanguage(config),
       area: Autocomplete.extractProductArea(config),
       pageSize: Configuration.extractAutocompleteProductCount(config)
-    }, config.autocomplete.overrides.products);
+    };
+
+    if (config.personalization.realTimeBiasing.autocomplete) {
+      request = Requests.realTimeBiasing(state, request);
+    }
+
+    return Requests.chain(config.autocomplete.defaults.products, request, config.autocomplete.overrides.products);
+  };
+
+  export const realTimeBiasing = (state: Store.State, request: Request): Request => {
+    const addedBiases = Personalization.convertBiasToSearch(state);
+
+    return {
+      ...request,
+      biasing: {
+        ...request.biasing,
+        biases: ((request.biasing ? request.biasing.biases : []) || []).concat(addedBiases),
```

Comment:
`(request.biasing && request.biasing.biases || []).concat(addedBiases)`, although I see that it was moved from `products.ts`, so up to you if you want to change it.